app/assets/stylesheets/incidents.scss

```diff
@@ -103,3 +103,18 @@ form .field .example{
     content: ' )';
   }
 }
+
+.map{
+  background-color: white;
+  height: 600px;
+  margin: 10px;
+  width: 600px;
```

Comment:
600x600px might look a little off on some screens. It's a bit small on my (giant) monitor, and too big on a phone.

I kinda like:

```css
.map {
  background-color: white;
  height: 50vh;
  width: 100%;
}
```